README.md

```diff
@@ -7,6 +7,7 @@ Here you'll find community contributions to the Gotify project.
 - [caronc/apprise](https://github.com/caronc/apprise) A notification library for almost all popular notification services.
 - [denisgolius/zabbix-to-gotify-alert](https://github.com/denisgolius/zabbix-to-gotify-alert) A guide for sending alerts from [Zabbix](https://www.zabbix.com/) to `gotify/server`.
 - [no-go/NotifyRelay](https://github.com/no-go/NotifyRelay) An Android app for relaying almost all Android messages to `gotify/server`. [F-Droid](https://f-droid.org/packages/click.dummer.notify_to_jabber/)
+- [ppacher/moziot-gotify-addon](https://github.com/ppacher/moziot-gotify-addon) A Mozilla WoT gateway addon to send notifications via `gotify`
```

Comment:
Can u add a period for consistency? Thanks.

Reply:
Done